.github/workflows/tf-checks.yml

```diff
@@ -0,0 +1,15 @@
+name: tf-checks
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+  workflow_dispatch:
+jobs:
+  basic:
+    uses: clouddrove/github-shared-workflows/.github/workflows/tf-checks.yml@1.2.2
+    with:
+      working_directory: './_example/'
+  complete:
+    uses: clouddrove/github-shared-workflows/.github/workflows/tf-checks.yml@1.2.2
+    with:
+      working_directory: './_example/'
```

Comment:
we have one example only so update according to it

Reply:
Sir I have done a changes please check if there are any mistake please let me know